core/server/src/main/java/alluxio/master/AlluxioMaster.java

```diff
@@ -442,6 +435,10 @@ protected void startServing(String startMessage, String stopMessage) {
   }
 
   protected void startServingWebServer() {
+    mWebServer = new MasterUIWebServer(ServiceType.MASTER_WEB.getServiceName(),
```

Comment:
the `stop` method shuts the web server down and sets `mWebServer` to null, so the `start` method needs to create the web server